### DISCUSSION TITLE
Implement Shared Photos to show under main Photos

### DISCUSSION THREAD

**Discussion by AndyXheli** [immich-app/immich#5013] [Feature Request]
### The feature

Hello I think i would be a nice idea if a user shares a photo or album that those photos show under the main root photos tab of users that is getting the share. . At the moment the only way for the user that you've shared to photos with would be to view them would be under the sharing tab 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by alextran1502** [comment]
Part of this is done in the next release (1.86) where you can show the media of your partners on your timeline

**Comment by AndyXheli** [comment]
Hi @alextran1502 I did see the change for 1.86 got a question. When i add the partner now I'm able to see all the photo i was looking more for just photos are are under the shared albums that would should under the timeline not everything dose that make sense

**Reply by alextran1502** [reply]
It does make sense and it doesn't work through way you described. All photos from your partners will show up on your timeline

**Reply by AndyXheli** [reply]
Correct I can see that if you add a partner you will see all their photos and vice versa. My question would be would there be a way to implement that if a user shares an album with another user all the content in that album which would show under the users timeline I think this would be really cool feature since most users share albums with one another then give access to their full Photos library. What do you think ?

**Reply by alextran1502** [reply]
Ah I see, I agree this would be a cool feature!

**Reply by AndyXheli** [reply]
@alextran1502 Should be create a new request on this so we can track it  or use this ?

**Reply by alextran1502** [reply]
We can use this one to track

**Reply by AndyXheli** [reply]
Perfect, Can you please reopen 😃

**Comment by aisbergde** [comment]
Partner sharing is not helpful, because it is all or nothing. If we could at least mark favorites in shared folders and they will be added, this would be fine. I don't want all assets automatically to be added. This is like spam. Only assets, I dedicate like, should be added. But liking (add to favorites) is not yet possible (at least on android mobile).

**Reply by Th0mC** [reply]
In your use case, I would just use a shared album and put here only the photos you want to share.

**Reply by aisbergde** [reply]
this is exactly, what I do. I just wanted to extrapolate the feature request from a partner context to a shared folder context. But maybe, this should be a different feature request.

**Comment by bo0tzz** [comment]
Closing as a dupe of #3984
